include/sempr/entity/example/PropertyTestEntity.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef SEMPR_ENTITY_EXAMPLE_PROPERTYTESTENTITY_HPP_
+#define SEMPR_ENTITY_EXAMPLE_PROPERTYTESTENTITY_HPP_
+
+#include <sempr/entity/SemanticEntity.hpp>
+#include <sempr/entity/example/Person.hpp>
+
+namespace sempr { namespace entity {
+
+#pragma db object
+class PropertyTestEntity : public SemanticEntity {
+    SEMPR_ENTITY
+public:
+    using Ptr = std::shared_ptr<PropertyTestEntity>;
+    PropertyTestEntity();
+
+    int intValue_;
+    float floatValue_;
+    double doubleValue_;
+    std::string stringValue_;
+    Person::Ptr entityValue_;
+
+    // RDFProperty<int> intValue_;
+    // RDFProperty<float> floatValue_;
+    // RDFProperty<double> doubleValue_;
+    // RDFProperty<std::string> stringValue_;
+    // RDFProperty<Person::Ptr> entityValue_;
+
+    // TODO test composite value
```

Comment:
Do you still like to test it? :-)

Reply:
Composite values are... not so straight forward. I'll leave this for a future extension, just like vectors. So the answer is: No. ^^

Reply:
Will you create a issue for this topic?